common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -890,6 +890,10 @@
   "feature.wynntils.quickCast.levelRequirementNotReached": "You need to reach %s %s before using this weapon.",
   "feature.wynntils.quickCast.name": "Quick Spell Cast",
   "feature.wynntils.quickCast.notAWeapon": "You can only quick-cast with a weapon!",
+  "feature.wynntils.quickCast.rightClickTickDelay.description": "How many ticks delay should there be before a new right-click is sent?",
```

Comment:
You should add here something like "clicks may not register sever-side when setting this lower than 2" because people probably don't know that there is a delay. Also, I think you should call it cooldown because you only delay it if there was a right-click before.

Reply:
okay, updated.